cluster/gke/config-default.sh

```diff
@@ -34,4 +34,5 @@ ELASTICSEARCH_LOGGING_REPLICAS=1
 #   none     - No cluster monitoring setup 
 #   influxdb - Heapster, InfluxDB, and Grafana 
 #   google   - Heapster, Google Cloud Monitoring, and Google Cloud Logging
-ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-none}"
+#   standalone     - Heapster only. Metrics available via Heapster REST API.
+ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-standalone}"
```

Comment:
We can do it in a separate PR.

On Tue, Jun 16, 2015 at 12:28 PM, Saad Ali <notifications@github.com> wrote:

> In cluster/gke/config-default.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9886#discussion_r32560600>
> :
>
> > @@ -34,4 +34,5 @@ ELASTICSEARCH_LOGGING_REPLICAS=1
> >  #   none     - No cluster monitoring setup
> >  #   influxdb - Heapster, InfluxDB, and Grafana
> >  #   google   - Heapster, Google Cloud Monitoring, and Google Cloud Logging
> > -ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-none}"
> > +#   standalone     - Heapster only. Metrics available via Heapster REST API.
> > +ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-standalone}"
>
> Worth removing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9886/files#r32560600>
> .
>